cyberbrain/value_stack.py

```diff
@@ -201,35 +306,39 @@ def _tos(self, n):
     def _push(self, *values):
```

Comment:
We can get rid of this `if..else`, it should now be safe to assume everything on the stack is a stack item instance, and that's the whole point of this migration: to formalize what is stored on the stack.

We should explictly convert things to `SymbolStackItem` or `CustomValueStackItem` before pushing them onto the stack, but I wonder if there's any place where we haven't done this (since `_placeholder` is already a `CustomValueStackItem`). We can probably just run tests to see if anything breaks.

Reply:
This will make the handler code messier, especially for exception handling with all the CustomValueStackItems but is doable

Reply:
I'm leaning towards doing it. It adds more clarity.

Reply:
Changed

Reply:
Thanks. Maybe add type annotation to `values`?

Reply:
This would need a union operator since we’re pushing both SymbolStackItems and CustomValueStackItems